test/lib/bamboo/adapters/mandrill_helper_test.exs

```diff
@@ -9,6 +9,44 @@ defmodule Bamboo.MandrillHelperTest do
     assert email.private.message_params == %{"track_links" => true}
   end
 
+  test "put_merge_vars/3 puts a list of merge_vars in private.merge_vars" do
+    users = [
+      %{
+        email: "user1@example.com",
+        full_name: "User 1"
+      },
+      %{
+        email: "user2@example.com",
+        full_name: "User 2"
+      }
+    ]
+
+    email = new_email
+    |> MandrillHelper.put_merge_vars(users, fn(user) -> %{ full_name: user.full_name } end)
```

Comment:
Could you indent this two spaces? Either that or put it on the same line and make the function multiline like this:

```elixir
email = MandrillHelper.put_merge_vars email, users, fn(user) ->
  %{full_name: user.full_name}
end
```